base/util/src/com/netscape/cmsutil/crypto/CryptoUtil.java

```diff
@@ -3111,20 +3111,30 @@ public static void importSharedSecretWithAES(byte[] wrappedSessionKey,byte[] wra
             throw new IOException("Shared secret " + sharedSecretNickname + " already exists");
         }
 
+        X509Certificate cert = cm.findCertByNickname(subsystemCertNickname);
+        logger.debug(method + " subsystemCert: " + cert);
```

Comment:
I know it's just the existing code being moved up, but doesn't findCertByNickname return null if nickname not found?  If so, would logger.debug cause NullPointerException?